egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json

```diff
@@ -446,7 +446,7 @@
       },
       {
         "module": "FSM",
-        "dateRefField": "Data.@timestamp",
+        "dateRefField": "Data.fsm.@timestamp",
         "requestQueryMap": "{\"wardId\" : \"Data.ward.name.keyword\", \r\n  \"district\" : \"Data.tenantData.city.districtCode\", \r\n\"tenantId\" : \"Data.tenantId\" \r\n}",
         "indexName": "fsm",
         "aggrQuery": "{\"aggs\":{\"AGGR\":{\"filter\":{\"bool\":{\"must_not\":[{\"term\":{\"Data.fsm.tenantId.keyword\":\"pb.testing\"}}]}},\"aggs\":{\"Applications Closed\":{\"filter\":{\"bool\":{\"must\":[{\"terms\":{\"Data.status.keyword\":[\"completed\"]}}]}},\"aggs\":{\"Count\":{\"value_count\":{\"field\":\"Data.@timestamp\"}}}}}}}}"
```

Comment:
Data.@timestamp should be changed to Data.fsm.@timestamp in thsi aggrQuery

Completed Status is CAPITAL Case COMPLETED and you can also consider application in status CITIZEN_FEEDBACK_PENDING also as completed, And not sure how Data.status.keyword will get the applicationStatus, it should be Data.fsm.applicationStatus